src/realm/CMakeLists.txt

```diff
@@ -0,0 +1,324 @@
+set(REALM_SOURCES
+    alloc.cpp
```

Comment:
Globbing generally isn't great, because new files aren't detected unless you manually rerun `cmake`. The manual has a soft warning against it.